### ISSUE TITLE
JSON linting for PRs

### ISSUE THREAD

**Issue by l-2-j** [neos-modding-group/neos-mod-manifest#74]
There's been quite a few PR's that had broken JSON or wrong indentation. Setting up GH actions to check for basic things like that and automatically request changes if there's issues should be done.

Also as a more complex possibility, since so many people use github links, we could do a regex check on the download/release urls and if they match Github urls but aren't pinned to a specific tag could leave a comment straight away about it on the PR.

**Comment by zkxs** [comment]
> i agree, we should also check that the download link matches the checksum automatically

Unfortunately that's not feasible, as the whole reason we have the checksum is because the download link has no guarantee of always serving the same file. While we could have automatic checksum validation as a *sanity* check, it wouldn't actually function as a *security* check, and I worry it would imply to auditors that they don't have to verify the checksum themselves.

**Comment by EIA485** [comment]
> and I worry it would imply to auditors that they don't have to verify the checksum themselves.
maybe we could explicitly have the bot state that "the checksum did not pass the sanity check"

**Comment by ljoonal** [comment]
Yeah IMO didn't make the bot ever say that it passed, only say if it _didn't_. And of course keep auditors informed that they still need to verify it locally too, and that the bot check is just a sanity check pass.

**Comment by EIA485** [comment]
the bot could also automatically resolve merge conflicts caused by multiple prs appending mods to the end of the list
